express-post-json/server.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+const app = express();
+
+const PORT = 8080;
+const GRADES_URL = '/api/grades';
+let nextID = 1;
+const grades = {};
+
+app.use(express.json());
+
+app.get(GRADES_URL, function getAllGrades(req, res) {
+  res.json(grades);
+});
+
```

Comment:
While the grades data is stored internally in an object, the GET response should still be an array of grades.